nature.md

```diff
@@ -1202,11 +1197,13 @@ Third party scripts, including advertising, may inject components that remove fo
 <li>602.3 (Support Docs)</li>
 </ul></td>
 <td>Does Not Support</td>
-<td>All components on the pages of this product supplied by us as part of dynamic templates use the same language defined in the HTML <code>lang</code> attribute, therefore Language of Parts is not applicable for the majority of this product.
+<td>Language of Parts is not applicable for the majority of this content. All first-party components on the pages of this product that are part of dynamic templates are written in English.
+<br><br>
+Editorial content or published papers may include text in languages other than English. 
 <br><br>
-Editorial content or published papers may include text in languages other than that defined in the HTML <code>lang</code> attribute. We currently have no method of confirming the inclusion - or not - of appropriate lang attributes for pieces of text in content that may use a different language to the main document. 
+Third party scripts may inject components with non-English text - for example a targeted advertisement or file presented in a file viewer (e.g. Figshare) may use a different language to the main document. These may not be marked up with a separate <code>lang</code> attribute.
 <br><br>
-Third party scripts may inject components that use a different language to the main document - for example a targeted advertisement or file presented in a file viewer (e.g. Figshare) may use the native language of the user. These may not be marked up with a separate <code>lang</code> attribute.
+We currently have no method of confirming the inclusion - or not - of appropriate lang attributes for pieces of text in content that may use a different language to the main document.
```

Comment:
Yeah, I'm not sure how to fully check this. Research content definitely includes the lang attribute at the `article` level and on alternate language headlines. I'm not sure about elsewhere in the content. I'm pretty certain magazine content doesn't have anything for this (to be honest I don't even know if we can find out what language the content was written in from the CMS)

Reply:
Interesting. Do you have an example of one of those alternate language headlines or a non-english research article? I thought Nature was English-only! Or is it just because Nature shares the same article template as SpringerLink, which does have multiple languages?

Reply:
Yeah - I suspect the research article stuff is only used on springer link.

Here's an example of a full article https://link.springer.com/article/10.1007/s41358-020-00247-y
Looking for another example where the article title gets translated. 

Here's a magazine example from nature italy where we get the language wrong https://www.nature.com/articles/d43978-021-00039-0

Reply:
https://link.springer.com/article/10.1007/BF00038917 is an example with mixed language content -  it looks like we're hopefully getting some things right - the article language is set to de, the alternate titles and summary section are set to en. But the sections we generate for author information, rights and permissions etc... don't get a lang attribute despite not matching the language of the article.

Reply:
@emsel5201 We'll update the VPAT to acknowledge that we do sometimes publish content on Nature in languages other than English, and that we don't correctly switch the lang attribute. There's a ticket to fix that open already, and we can update this again once there's something to report. 

Thanks @moddular!

Reply:
I've been working on this today :) 

We've deployed a change to oscar-magazine-publisher that should use the language set by the CMS when the XML gets published. 

It's likely quite far from perfect - it won't handle mixed languages at all - and because it's a change to the publishing process existing content would need to be republished to pick it up. But I think it's at least better than whacking lang="en" on everything.

https://github.com/springernature/oscar/commit/67c02794cc25b87931315f6bba7679028e5a2eef 

From here we need to update the renderer to get the language from the XML and add it into the page.

Reply:
This lets us take SpringerLink from "does not support" to "partially supports" for Language of Parts 🧐 🎉